### PR TITLE
Features/early bound entities

### PR DESCRIPTION
**Pull request by StephBrouil**
- Replaced CrmSvcUtil with Early Bound Generator
- Configured Early Bound Generator
- Configured type definitions to be in separate files

### CODE REVIEW

**Comment by joshcampbell191** [comment]
@pierluc-ssoucy There are quite a few changes in this PR however most of them are due to auto-generated files. 

In order to simplify the PR, you can focus on the following:
- README.md
- unpack-commit-solution.yml
- lgtm.yml
- crmsvcutil.exe.config 
- DLaB.EarlyBoundGenerator.DefaultSettings.xml
- generate_actions.ps1 
- generate_entities.ps1 
- generate_optionsets.ps1 
- get_XrmDefinitelyTyped.ps1 
- package.json
- tsconfig.json

**Review by Sargatos — COMMENTED** [review]
I don't have much comments. Maybe we should add a bit more information in the readme, but else it looks good.

File: src/Compliance.Solution/Solution/Entities/opc_event/Entity.xml [hunk 1]
```diff
@@ -2871,7 +2871,7 @@
           <IsLocalizable>0</IsLocalizable>
           <Format>text</Format>
           <MaxLength>204</MaxLength>
-          <Length>1600</Length>
+          <Length>800</Length>
```
[the remark this answers is not in the document]
Comment (Sargatos) [reply src/Compliance.Solution/Solution/Entities/opc_event/Entity.xml]: Puisque vous y avez pas touché, on va le laisser comme ca et s'il le faut on revertera cette partie seulement.

**Review by Sargatos — APPROVED** [review]
I think it's all good.

**Pull request merged**